### DISCUSSION TITLE
Mods vs Scripts?

### DISCUSSION THREAD

**Discussion by pulynerain** [malforge/mdk2#25] [General]
I got incredible value from the original MDK and have now begun using MDK2.  Is there support for creating Mods vs MDK2 yet vs scripts?  I need to create a text surface app which must be a mod as it requires server components.  Thank you very much!

**Comment by malware-dev** [comment]
Not yet. My focus must be to complete the ingame script support first, since that's the primary function that MDK was made for. But it is within the plans.

**Comment by malware-dev** [comment]
There is now preliminary mod support in MDK2. Sorry it took so long 😄
